src/sql/run.py

```diff
@@ -170,7 +173,28 @@ def DataFrame(self, payload):
         "Returns a Pandas DataFrame instance built from the result set."
         import pandas as pd
 
+        "create a method to check for urls"
```

Comment:
this should be a comment, not a string

```suggestion
        # create a method to check for urls"
```

please also apply it to the other comments

Reply:
Made the changes